lib/advisor/core/question_finder.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Advisor.Core.QuestionFinder do
+  alias Advisor.Core.Question
+  alias Advisor.Repo
+  import Ecto.Query
+
+  def find_all(ids) do
+    query = from q in Question,
+      where: q.id in ^ids
```

Comment:
This should fit in a line. And maybe we can online query? I have a hunch this can be applied to other queries...